lib/std/http.zig

```diff
@@ -1,8 +1,251 @@
-const std = @import("std.zig");
+pub const Client = @import("http/Client.zig");
+
+/// https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods
+/// https://datatracker.ietf.org/doc/html/rfc7231#section-4 Initial definiton
+/// https://datatracker.ietf.org/doc/html/rfc5789#section-2 PATCH
+pub const Method = enum {
+    GET,
+    HEAD,
+    POST,
+    PUT,
+    DELETE,
+    CONNECT,
+    OPTIONS,
+    TRACE,
+    PATCH,
```

Comment:
remark

If it's considered widespread/mature enough, `QUERY` might be added.
ref https://datatracker.ietf.org/doc/draft-ietf-httpbis-safe-method-w-body/